test/parallel/test-http2-generic-streams-sendfile.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+const common = require('../common');
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+const assert = require('assert');
+const http2 = require('http2');
+const fs = require('fs');
+const MakeDuplexPair = require('../common/duplexpair');
+
+{
+  const testData = '<h1>Hello World</h1>';
+  const server = http2.createServer();
+  server.on('stream', common.mustCall((stream, headers) => {
+    stream.respondWithFile(__filename);
+  }));
+
+  const { clientSide, serverSide } = MakeDuplexPair();
+  server.emit('connection', serverSide);
+
+  const client = http2.connect('http://localhost:80', {
+    createConnection: common.mustCall(() => clientSide)
+  });
+
+  const req = client.request({ ':path': '/' });
+
+  req.on('response', common.mustCall((headers) => {
+    assert.strictEqual(headers[':status'], 200);
+  }));
+
+  req.setEncoding('utf8');
+  let data = '';
+  req.on('data', (chunk) => {
```

Comment:
Nit: ideally a mustCall?

Reply:
This is just accumulating the data, and that’s checked in an `assert` (inside a `mustCall`) anyway

Reply:
mustCall's on data events can be problematic. If anything, a `mustCallAtLeast` would be ok.

Reply:
Sorry, this was strictly about the consistency of these calls. In other tests with this PR, data events have `mustCall`. It sets a good example for newer contributors when there's consistency to where we `mustCall` and where we don't. Or we could leave a comment explaining why not.

But as I mentioned, this is just a very minor nit. :)

Reply:
Fwiw, the other call is somewhat intentionally checking that the data is sent through in a single chunk :) I’ve added 2a666de1940b as a comment for that